tests/PHPUnit/Unit/RankingQueryTest.php

```diff
@@ -82,6 +82,114 @@ public function testBasic()
+                    END AS counter,
+                    CASE 
+                        WHEN `label` IS NULL AND `url` IS NULL THEN -1
+                        WHEN `label` IS NULL AND @counterRollup = 11 THEN 11
+                        WHEN `label` IS NULL THEN @counterRollup := @counterRollup + 1
+                        WHEN `url` IS NULL AND @counterRollup = 11 THEN 11
+                        WHEN `url` IS NULL THEN @counterRollup := @counterRollup + 1
+                        ELSE 0
+                    END AS counterRollup,
+                    `column`,
+                    `columnSum`
+                FROM
+                    ( SELECT @counter:=0 ) initCounter,
+                    ( SELECT @counterRollup:=0 ) initCounterRollup,
+                    ( SELECT label, url, column, columnSum FROM myTable) actualQuery
+            ) AS withCounter
+            GROUP BY counter, counterRollup
+        ";
+
+        if (!Schema::getInstance()->supportsSortingInSubquery()) {
+            $expected = "
+                SELECT
+                    CASE
+                        WHEN counterRollup = 11 THEN 'Others'
+                        WHEN counterRollup > 0 THEN `label`
+                        WHEN counter = 11 THEN 'Others'
+                        ELSE `label`
+                    END AS `label`,
+                    CASE
+                        WHEN counterRollup = 11 THEN NULL
+                        WHEN counterRollup > 0 THEN `url`
+                        WHEN counter = 11 THEN 'Others'
+                        ELSE `url`
+                    END AS `url`,
+                    `column`,
+                    sum(`columnSum`) AS `columnSum`
+                FROM (
+                    SELECT 
+                        `label`, `url`,
+                        CASE 
+                            WHEN `label` IS NULL THEN -1
+                            WHEN `url` IS NULL THEN -1
+                            WHEN @counter = 11 THEN 11
+                            ELSE @counter:=@counter+1
+                        END AS counter,
+                        CASE
+                            WHEN `label` IS NULL AND `url` IS NULL THEN -1
+                            WHEN `label` IS NULL AND @counterRollup = 11 THEN 11
+                            WHEN `label` IS NULL THEN @counterRollup := @counterRollup + 1
+                            WHEN `url` IS NULL AND @counterRollup = 11 THEN 11
+                            WHEN `url` IS NULL THEN @counterRollup := @counterRollup + 1
+                            ELSE 0
+                        END AS counterRollup,
+                        `column`,
+                        `columnSum`
+                    FROM
+                        ( SELECT @counter:=0 ) initCounter,
+                        ( SELECT @counterRollup:=0 ) initCounterRollup,
+                        ( SELECT label, url, column, columnSum FROM myTable LIMIT 18446744073709551615 )
+actualQuery
```

Comment:
```suggestion
                        ( SELECT label, url, column, columnSum FROM myTable LIMIT 18446744073709551615 ) actualQuery
```

The result comparison won't care about that change, but the line break looks a bit off.

Reply:
Fixed.